pkg/ui/ccl/src/views/reports/containers/stores/encryption.tsx

```diff
@@ -1,10 +1,24 @@
 import React from "react";
 
 import * as protos from "src/js/protos";
+import _ from "lodash";
+import Long from "long";
+import moment from "moment";
 import { EncryptionStatusProps } from "oss/src/views/reports/containers/stores/encryption";
+import { Bytes } from "src/util/format";
+
+const dateFormat = "Y-MM-DD HH:mm:ss Z";
```

Comment:
Suggest omitting the time zone here.

Also maybe leverage the helpers in `src/views/reports/containers/range/print` (which could probably be moved to `src/util`)